javascript/sdk/lib/types.ts

```diff
@@ -240,3 +242,13 @@ export interface ProbeActivity {
   state: "PROTECTED" | "UNPROTECTED" | "ERROR";
   tags: string[];
 }
+
+export interface CreateRecommendation {
+  title: string;
+  description: string;
```

Comment:
`decision: int`

Reply:
Add it right now so that you don't have to add it later :)

Reply:
oh this is to create a recommendation not decide on it. Unless it will be the same thing soon :)

copied from:
https://github.com/preludeorg/libraries/blob/master/python/sdk/prelude_sdk/controllers/detect_controller.py#L117

Reply:
It will be the same api endpoint

Reply:
You need an id field too in there